lib/cylc/task_pool.py

```diff
@@ -966,23 +966,20 @@ def process_queued_task_event_handlers(self):
                 # Set timer if timeout is None.
                 if not try_state.is_timeout_set():
                     if try_state.next() is None:
-                        itask.log(ERROR, "%s failed:\n\t%s" % (
-                            key,
-                            try_state.ctx))
+                        itask.log(ERROR, "%s failed" % (key))
```

Comment:
```
Traceback (most recent call last):
  File "/home/oliverh/cylc/cylc.git/lib/cylc/run.py", line 82, in main
    server.run()
  File "/home/oliverh/cylc/cylc.git/lib/cylc/scheduler.py", line 1001, in run
    self.pool.process_queued_task_event_handlers()
  File "/home/oliverh/cylc/cylc.git/lib/cylc/task_pool.py", line 970, in process_queued_task_event_handlers
    itask.log(ERROR, "%s failed:" % (key))
TypeError: not all arguments converted during string formatting
```
This is odd because the original code successfully puts `key` in the string, but to avoid the error I need an explicit string conversion here: `% str(key)`.